packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -162,6 +158,17 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
   let showActions = tagState.showCollapseButton || (actionLabel && onAction);
   let isEmpty = state.collection.size === 0;
 
+  let containerStyle = useMemo(() => {
+    if (maxRows == null || !isCollapsed || !tagsRef.current) {
+      return undefined;
+    }
+    let tag = tagsRef.current.children[0];
+    let tagStyle = window.getComputedStyle(tag);
+    let maxHeight = (tag.getBoundingClientRect().height + parseInt(tagStyle.marginTop, 10) + parseInt(tagStyle.marginBottom, 10)) * maxRows;
```

Comment:
this is probably only two values, mobile and desktop, right? can we hard code them?

Reply:
I suppose if we really wanted to optimize, we could memoize these values and only refresh them if theme changes.

Reply:
lets do that